### PR TITLE
add initials util and tests

### PR DESCRIPTION
**Pull request by crmorford**

### CODE REVIEW

File: test/util/string.js [hunk 1]
```diff
@@ -81,4 +82,31 @@ describe(`toSentenceCase`, function() {
...
+describe(`nameToInitials`, function() {
+  it(`returns one initial when the name is only one word`, function() {
+    expect(nameToInitials(`Cassie`)).to.eql(`C`);
+  });
+
+  it(`returns both initials for a first name and last name`, function() {
+    expect(nameToInitials(`Cassie Morford`)).to.eql(`CM`);
+  });
+
+  it(`returns two initials for a first, middle and last name`, function() {
```
Comment (jalack) [inline test/util/string.js]: oxford comma? 😉

File: lib/util/string.js [hunk 2]
```diff
@@ -58,3 +58,12 @@ export function toSentenceCase(string) {
   const str = (string || ``).toString();
   return str.charAt(0).toUpperCase() + str.substr(1).toLowerCase();
 }
+
+export function nameToInitials(name) {
+  if (!name) {
+    return ``;
+  }
+  // return a maximum of 2 initials for a user
+  let initials = name.split(` `).map(s => s[0]).join(``).toUpperCase();
```
Comment (jalack) [inline lib/util/string.js]: nit nit: you can make this `const`

Comment (crmorford) [inline lib/util/string.js]: oh tru

**Comment by tdumitrescu** [comment]
did a little cleanup and published in v0.55.3

**Pull request merged**